digitalpublicgoods/pharmacovigilance-electronic-reporting-tool.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "Pharmacovigilance Electronic Reporting Tool",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://pv.pharmacyboardkenya.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "Installation guide - https://github.com/IntelliSOFT-Consulting/pvers/blob/master/README.md",
+      "https://docs.google.com/document/d/1LjkPOKrK-4O_PBb4GRfx61I2_0zfesD_5X1y9aw34dQ/edit#heading=h.6nlwssopwwv3"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Data is extracted via PDF/PNG/CSV for individual indicators (e.g gender)"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "Kenya Standards and Guidelines for E-Health Systems Interoperability: https://pdf.usaid.gov/pdf_docs/PA00TB28.pdf",
+      "The Kenya Data Protection Act 2019: https://www.odpc.go.ke/dpa-act/",
+      "General Data Protection Regulation"
+    ],
+    "adherenceSteps": [
+      "Privacy Policy:https://web.pharmacyboardkenya.org/terms-and-conditions/",
+      "Cookie's consent: https://web.pharmacyboardkenya.org/pharmacovigilance/"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "File exchange formats as ICH E2B CIOMS",
+      "Medical terminologies such MedDRA"
+    ],
+    "evidenceStandardSupport": [
+      "https://web.pharmacyboardkenya.org/pharmacovigilance/"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Design for scale",
+      "Be data driven ",
+      "Reuse and improve",
+      "Be collaborative"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "It has inbuilt flags to identify serious cases for rapid response. It also provides a communication platform that allows for clarification on reported cases. There are also reminders sent to reporters for any unsubmitted reports."
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "No",
```

Comment:
Can we clarify if they collect PII?
This seems to part of the platform features where they collect things like name, email, country, etc: https://pv.pharmacyboardkenya.org/padrs/add